templates/default/event/eventinfoshort.html

```diff
@@ -84,7 +84,7 @@
             {icon name="status"}
         </TD>
         <TD class="bold">
-            {trans("Status:")}
+            {trans("Availability:")}
```

Comment:
Dostępność - wg mnie nie pasuje do tego co określa.

Reply:
Zaproponujesz coś lepiej? Dostępność: publiczna/prywatna wg mnie wydaje się ok.
Synonimy jakie również pasują to "jawność" i "osiągalność".

Obecnie nazwa "status" jest nieprecyzyjna, w zasadzie też jest mylona przez użytkowników ze statusem zdarzenia zamknięte/otwarte.

Reply:
Masz rację - zapomniałem, że status tutaj oznacza właśnie "dostępność"!